pyproject.toml

```diff
@@ -25,8 +25,8 @@ requires-python = ">=3.8"
 dependencies = [
     "scipy",
     "numpy",
-    "pandas~=1.5.0",
-    "matplotlib>=3.5.0",
+    "pandas~=2.0.0",
```

Comment:
I used ~= to force us to actually check that e.g. pandas 2.1 doesn't break anything when it comes along. I can change if you would prefer >=?

Reply:
I think `>=` would be better. Should pandas 2.1 indeed break something, it will pop up in our daily scheduled CI checks of the master branch with `>=`. When the master branch fails from one day to the next it is typically down to a new matplotlib or pandas version. With `~=`, on the other hand, we won't notice it in the daily checks.